docs/docsite/rst/locales/ja/LC_MESSAGES/community.po

```diff
@@ -147,2604 +155,6369 @@ msgid "Policy violations"
+msgstr "\\*\\* 統合テストは必須であり、ユニットテストは歓迎されますが、AWS コレクションには必須ではありません。これに対する例外は、外部要件により統合テストが永続的に実行できない場合に生じます。この例として、AWS Direct Connect があります。これは、ネットワークピアリング接続を確立しないと、このサービスを機能的にテストできないためです。したがって、AWS Direct Connect と相互作用するモジュールにはユニットテストが必要です。``amazon.aws`` への例外は Red Hat によって承認され、``community.aws`` への例外は AWS コミュニティーによって承認される必要があります。"
+
+#: ../../rst/community/contributing_maintained_collections.rst:262
+msgid "\\*\\*\\* ``ansible.netcommon`` contains all foundational components for enabling many network and security :ref:`platform <platform_options>` collections. It contains all connection and filter plugins required, and installs as a dependency when you install the platform collection."
+msgstr "\\*\\*\\* ``ansible.netcommon`` には、多くのネットワークおよびセキュリティーの :ref:`プラットフォーム <platform_options>` コレクションを有効にするための基本コンポーネントが含まれています。これには必要なすべての接続およびフィルタープラグインが含まれ、プラットフォームコレクションのインストール時に依存関係としてインストールされます。"
+
+#: ../../rst/community/contributing_maintained_collections.rst:264
+msgid "\\*\\*\\*\\* Unit tests for Windows PowerShell modules are an exception to testing, but unit tests are valid and required for the remainder of the collection, including Ansible-side plugins."
+msgstr "\\*\\*\\*\\* Windows PowerShell モジュールのユニットテストは、テストの例外ですが、ユニットテストは、Ansible 側のプラグインを含め、コレクションの残りの部分に有効かつ必要になります。"
+
+#: ../../rst/community/contributing_maintained_collections.rst:270
+msgid "Deciding where your contribution belongs"
+msgstr "貢献が所属する場所の決定"
+
+#: ../../rst/community/contributing_maintained_collections.rst:272
+msgid "We welcome contributions to Ansible-maintained collections. Because these collections are part of a downstream supported Red Hat product, the criteria for contribution, testing, and release may be higher than other community collections. The related community collections (such as ``community.general`` and ``community.network``) have less-stringent requirements and are a great place for new functionality that may become part of the Ansible-maintained collection in a future release."
+msgstr "Ansible で管理されているコレクションへの貢献を歓迎します。これらのコレクションは、ダウンストリームでサポートされている Red Hat 製品の一部であるため、貢献、テスト、およびリリースの基準は、他のコミュニティーコレクションよりも高い場合があります。関連するコミュニティーコレクション (``community.general`` および ``community.network`` など) の要件はそれほど厳しくなく、将来のリリースで Ansible が管理するコレクションの一部になる可能性がある新しい機能に最適な場所です。"
+
+#: ../../rst/community/contributing_maintained_collections.rst:274
+msgid "The following scenarios use the ``arista.eos`` to help explain when to contribute to the Ansible-maintained collection, and when to propose your change or idea to the related community collection:"
+msgstr "以下のシナリオでは、``arista.eos`` を使用して Ansible が管理しているコレクションに貢献するタイミングや、関連するコミュニティーコレクションに変更や考えを提案するタイミングを説明しています。"
+
+#: ../../rst/community/contributing_maintained_collections.rst:277
+msgid "You want to fix a problem in the ``arista.eos`` Ansible-maintained collection. Create the PR directly in the `arista.eos collection GitHub repository <https://github.com/ansible-collections/arista.eos>`_. Apply all the :ref:`merge requirements
```

Comment:
Not sure why this wasn't translated?

Reply:
Not sure what you mean here. This looks translated. Was that the line you were refering to @samccann ?

```
msgid "You want to fix a problem in the ``arista.eos`` Ansible-maintained collection. Create the PR directly in the `arista.eos collection GitHub repository <https://github.com/ansible-collections/arista.eos>`_. Apply all the :ref:`merge requirements <ansible_collection_merge_requirements>`."
msgstr "Ansible が管理する ``arista.eos`` コレクションの問題を修正します。PR を `arista.eos コレクションの GitHub リポジトリー <https://github.com/ansible-collections/arista.eos>`_ に直接作成します。:ref:`マージ要件 <ansible_collection_merge_requirements>` をすべて適用します。"
```

Reply:
Hmm not sure why it's not picking the correct line. I'm talking about line 2960 
```
#: ../../rst/community/contributor_license_agreement.rst:7
msgid "By contributing you agree that these contributions are your own (or approved by your employer) and you grant a full, complete, irrevocable copyright license to all users and developers of the project, present and future, pursuant to the license of the project."
msgstr "By contributing you agree that these contributions are your own (or approved by your employer) and you grant a full, complete, irrevocable copyright license to all users and developers of the project, present and future, pursuant to the license of the project."
```